dwr-hecdssvue/build.gradle

```diff
@@ -0,0 +1,81 @@
+plugins{
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+
+    // The following plugin is used to build the OSGI bundle
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+configurations {
+    eclipseLibs
+    implementation.extendsFrom(eclipseLibs)
+}
+
+dependencies {
+    //additional dependencies
+    implementation libs.hec.monolith
```

Comment:
These dependencies should be coming from third-party right?

Reply:
Since the hec jars are provided through an hec-dependencies module, no these dependencies are not set to come from third-party at this point.

Reply:
The last design discussion was that all third-party libraries would come from the third-party module. Did that decision get reversed?

Reply:
We will verify this consolidation with Hao and merge the hec-dependencies module into third-party after the installer is building successfully. Tracking that under this issue: https://github.com/CentralValleyModeling/wrims/issues/152